Cheers.java

```diff
@@ -1,6 +1,32 @@
-// Prints a crowd cheering output.
+
 public class Cheers {
         public static void main(String[] args) {
-	    // Replace this comment with your code
+	    String cheer = args[0];
+        int times = Integer.parseInt(args[1]);
+        String bigLetters = "AEFHILMNORSX";
+        String smallLetters = "aefhilmnorsx";
+        String finalCheer = "";
+            int i = 0;
+            while  (i < cheer.length() ) {
```

Comment:
1.  This is by definition a for loop.. 
2. declare i in for line, no usage outside the loop